.github/workflows/c.yaml

```diff
@@ -0,0 +1,35 @@
+name: C
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    paths:
+      - "c/**"
+      - .github/workflows/c.yaml
+jobs:
+  coverage:
+    name: Test
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Setup
+        run: |
+          sudo apt-get -qq update
+          sudo apt-get -qq install lcov
+
+      - name: Run tests
+        run: make test-c
+        env:
+          CMAKEFLAGS: -DCODE_COVERAGE=ON -DCMAKE_BUILD_TYPE=Debug
+
+      - name: Upload coverage to codecov.io
+        run: |
+          bash <(curl -s https://codecov.io/bash) -s c/build || echo "Codecov did not collect coverage reports";
```

Comment:
IIRC, the bash codecov usually doesnt set an exit status, so the `|| ..` here is useless.

This upload isnt working; see https://github.com/swift-nav/libsbp/runs/2124966802#step:5:530

It seems codecov hasnt been working for a while in this repo, so perhaps that could be another task to get them all working in github actions after this PR.

Reply:
It seems codecov<->github-integration isnt enabled for this repo.  It is possible to send reports without that integration being enabled.